tests/helpers.py

```diff
@@ -17,7 +18,9 @@ def assert_key_value(annex, key, value):
 
 def get_upload_info(client, key, **kwargs):
     response = client.get('/upload_info/{}'.format(key), **kwargs)
-    return json.loads(response.get_data(as_text=True))
+    return json.loads(
+        response.get_data(as_text=True), object_pairs_hook=OrderedDict,
```

Comment:
wait why do you care about `OrderedDict` here? there shouldn't be any more `dict`s in the output

Reply:
I think that's a mistake. I'm not sure why it's failing on python 2.7 and 3.5. Looks like the keys aren't in the expected order

Reply:
oh, right. let's kill support for <3.6

Reply:
the dict coming back from boto won't have the right order in <3.6

Reply:
Reverted that change. It looks like the `generate_presigned_post` call is returning an unexpected order. Otherwise, I'm unsure why the data are out of order in older python versions